tests/appsec/iast/aspects/test_re_aspects.py

```diff
@@ -491,3 +492,50 @@ def test_re_finditer_aspect_not_tainted():
     assert isinstance(res_iterator, typing.Iterator)
     for i in res_iterator:
         assert not is_pyobject_tainted(i)
+
+
+def test_re_match_getitem_aspect_tainted_string_re_object():
```

Comment:
I think a test to check bytes too are worthy

Reply:
Done!